### DISCUSSION TITLE
Homebrew on macOS Big Sur

### DISCUSSION THREAD

**Discussion by TanMasked** [Homebrew/discussions#5238] [Getting started]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

Apparently, the newest version of Homebrew (I'm trying to install via the .pkg installer) doesn't work on macOS Big Sur (my current macOS operating system.) Is there a way to access older versions of Homebrew?

**Comment by SMillerDev** [comment]
> Is there a way to access older versions of Homebrew?

Not if you don't have it installed yet.

**Reply by TanMasked** [reply]
<img width="417" alt="Screenshot 2024-03-28 at 09 23 49" src="https://github.com/Homebrew/discussions/assets/165265223/d1811a76-66c1-4f46-813c-c194b7e8b12f">

I can't even install it.
